tool/main.cpp

```diff
@@ -85,57 +85,79 @@ int main(int argc, const char* argv[]) {
     // Quantization parameters.
     ptg_quantization_parameters quantizationParameters;
 
+    // Tracing parameters.
+    ptg_tracing_parameters tracingParameters;
```

Comment:
This is never used in the generation_parameters struct.

Reply:
Fixed